.github/workflows/ci.yml

```diff
@@ -1,20 +0,0 @@
-name: CI
-

```

Comment:
Why this file was deleted? It should be there and it should have the same `3.1.100-preview3-014645` SDK version.

Reply:
Sorry, was having trouble doing it on mobile. I'll get it all fixed

Reply:
No rush, take your time.